docs/sources/panels/visualizations/graph-panel.md

```diff
@@ -0,0 +1,127 @@
++++
+title = "Graph panel"
+keywords = ["grafana", "graph panel", "documentation", "guide", "graph"]
+type = "docs"
+aliases = ["/docs/grafana/latest/reference/graph/", "/docs/grafana/latest/features/panels/graph/"]
+[menu.docs]
+parent = "visualizations"
+weight = 100
+draft = "true"
++++
+
+# Graph panel
+
+This visualization is the most-used in the Grafana ecosystem. It can render as a line, a path of dots, or a series of bars. This type of graph is versatile enough to display almost any time-series data.
+
+## Data and field options
+
+Graph visualizations allow you to apply:
+- Data transformations
+- Alerts - This is the only type of visualization that allows you to set alerts. For more information, see [LINK TO UPDATED ALERTING TOPIC]
+- [Thresholds]({{< relref "../thresholds.md" >}})
```

Comment:
Graph visualization has no data transforms except the X axes mode for histogram is a sort of transform. 

We cannot link to the generic thresholds topic here as Graph is not using the new field options thresholds but thresholds unique to the graph.

Reply:
But graph of course supports the new Transform feature (that all visualizations support),

Reply:
Regarding the Thresholds, I included graph instructions there. We can then delete them when the new Thresholds are added to this visualizatio and we won't even need to change the link.

"Data transformations" will eventually link to the documentation for the new Transform tab. It just hasn't been written yet.